src/System.Private.ServiceModel/src/System/ServiceModel/Channels/MessageContent.cs

```diff
@@ -15,9 +15,11 @@ namespace System.ServiceModel.Channels
 {
     public abstract class MessageContent : HttpContent
     {
+#pragma warning disable 3008	// not CLS-compliant
         protected Message _message;
         protected MessageEncoder _messageEncoder;
         protected Stream _stream = null;
+#pragma warning restore 3008	// not CLS-compliant
```

Comment:
If they are left as not CLS compliant, this simply means that those fields can't be accessed by code written in some other languages which only support CLS compliant assemblies. As this is only public to enable writing unit tests and isn't exposed on the contract, in practice this non-CLS compliance doesn't negatively affect any scenario.